CHANGELOG.md

```diff
@@ -7,14 +7,16 @@ This file is used to list changes made in each version of the aws-parallelcluste
 ------
 
 **CHANGES**
-
 - Drop support for SGE and Torque schedulers.
 - Use tags prefix `parallelcluster:`.
 - Run Slurm command `scontrol` with sudo because clustermgtd is run as cluster admin user (not root).
 - Implement `computemgtd` self-termination via `shutdown` command instead of calling TerminateInstances.
 - Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
   in case recurrent failures are encountered when provisioning nodes.
 
+**BUG FIXES**
+- Slurm: fix issue that prevented powering-up nodes to be correctly reset after a stop and start of the cluster.
+
```

Comment:
not sure I'd include this in the 3.0.0 changelog. I'd rather add here the changelogs for 2.11.x that are published before we release 3.0.0

Reply:
Moved changelog from 2.11 branch